src/oscar/apps/order/admin.py

```diff
@@ -81,3 +82,4 @@ class OrderDiscountAdmin(admin.ModelAdmin):
 admin.site.register(OrderDiscount, OrderDiscountAdmin)
 admin.site.register(CommunicationEvent)
 admin.site.register(BillingAddress)
+admin.site.register(Surcharge)
```

Comment:
We should use a custom admin class here and set `raw_id_fields` for things like Order - otherwise on a site with thousands of orders this becomes unusable.